Formula/rio.rb

```diff
@@ -6,10 +6,16 @@ class Rio < Formula
   license "MIT"
   head "https://github.com/raphamorim/rio.git", branch: "main"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
you dont need, the default should just works

Reply:
ah ok, removed

Reply:
I do see the issue, you have a mis-tag

<img width="548" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/1580956/548e38a7-93ba-49ba-b45a-e5c4fbfe4469">

Reply:
The issue for livecheck?

Reply:
Would delete the tag fix it?

Reply:
yes, it would

Reply:
this is the default livecheck reporting

```
$ brew livecheck rio
rio: 0.0.7 ==> 0.0.61
```

Reply:
deleted